bin/action_subscriber

```diff
@@ -11,6 +11,7 @@ module ActionSubscriber
     class_option :mode
     class_option :host
     class_option :hosts
+    class_option :prefetch, :type => :numeric, :desc => "how many messages to hold in the local queue in subscribe mode"
```

Comment:
This is a feature I would expect to be in place. I think it's a simple enough change to include in this PR. :+1: